sdk/storage/azure-storage-blob/src/test/java/com/azure/android/storage/blob/StorageBlobClientTest.java

```diff
@@ -85,14 +90,15 @@ public void getBlobsInPage() {
 
         mockWebServer.enqueue(mockResponse);
 
-        List<BlobItem> blobItems = storageBlobClient.getBlobsInPage(null,
+        BlobsPage blobsPage = storageBlobClient.getBlobsInPage(null,
             "testContainer",
             null);
 
         // Then a list containing the details of the blobs will be returned by the service and converted to BlobItem
         // objects by the client.
-        assertNotEquals(0, blobItems.size());
-        assertEquals("test.jpg", blobItems.get(0).getName());
+        assertNotEquals(0, blobsPage.getItems()
+            .size());
```

Comment:
nit: I don't think we need to have a new line here.
```suggestion
        assertNotEquals(0, blobsPage.getItems().size());
```